test-requirements.txt

```diff
@@ -2,4 +2,8 @@
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
 doc8
-flake8
\ No newline at end of file
+flake8>=5.0.4 # MIT
```

Comment:
`>=5.0` is more then enough. It is not recommended to pin to the patch version

Reply:
done